vm/src/builtins.rs

```diff
@@ -766,6 +766,7 @@ pub fn make_module(ctx: &PyContext) -> PyObjectRef {
 pub fn builtin_build_class_(vm: &VirtualMachine, mut args: PyFuncArgs) -> PyResult {
     let function = args.shift();
     let name_arg = args.shift();
+    let doc = args.shift();
```

Comment:
See my other comments, this code can be removed if you solve the set attr in the compilation phase.